pkg/security/probe/probe_windows.go

```diff
@@ -159,7 +159,9 @@ func (p *WindowsProbe) initEtwFIM() error {
 	if err != nil {
 		return err
 	}
-	p.fimSession, err = etwcomp.NewSession(etwSessionName)
+	p.fimSession, err = etwcomp.NewSession(etwSessionName, func(cfg *etw.SessionConfiguration) {
```

Comment:
it seems from the implementation of NewSession that we could pass nil instead of an empty body function here, but probably just a taste question

Reply:
true.
It's a derivative of another branch where I had several, unrelated changes for doing the perf testing.

My orig implementation couldn't handle a `nil`.  And, my orig implementation actually filled in something here.  But in order to split out into smaller PRs, this PR is only to add the ability to set that config.